firefox-ios/Client/Frontend/Browser/FindInPageBar.swift

```diff
@@ -50,21 +50,27 @@ class FindInPageBar: UIView, ThemeApplicable {
     }
 
     private lazy var previousButton: UIButton = .build { button in
-        button.setImage(UIImage(named: StandardImageIdentifiers.Large.chevronUp), for: .normal)
+        button.setImage(
+            UIImage(named: StandardImageIdentifiers.Large.chevronUp)?.withRenderingMode(.alwaysTemplate),
+            for: .normal)
         button.accessibilityLabel = .FindInPagePreviousAccessibilityLabel
         button.addTarget(self, action: #selector(self.didFindPrevious), for: .touchUpInside)
         button.accessibilityIdentifier = AccessibilityIdentifiers.FindInPage.findPreviousButton
     }
-
+    
```

Comment:
```suggestion
```

@dicarobinho it seems like this PR has lint issue

I suggest installing `swiftlint` using homebrew and running it before pushing any code 

<img width="591" alt="image" src="https://github.com/mozilla-mobile/firefox-ios/assets/8919439/0607fde3-38d4-458b-8bc6-33009da3820b">

Reply:
@nbhasin2 is seems like on line 68 is the same lint issue. May you have please a quick change suggestion like this one for that line?